cg/meta/workflow/nf_analysis.py

```diff
@@ -4,19 +4,29 @@
 from typing import Any
 
 from cg.constants import Pipeline
-from cg.constants.constants import FileExtensions, FileFormat, WorkflowManager
+from cg.constants.constants import FileExtensions, FileFormat, WorkflowManager, MultiQC
 from cg.constants.nextflow import NFX_WORK_DIR
-from cg.exc import CgError
-from cg.io.controller import WriteFile
 from cg.io.yaml import write_yaml_nextflow_style
 from cg.meta.workflow.analysis import AnalysisAPI
 from cg.meta.workflow.nf_handlers import NextflowHandler, NfTowerHandler
 from cg.models.cg_config import CGConfig
 from cg.models.fastq import FastqFileMeta
 from cg.models.nf_analysis import FileDeliverable, PipelineDeliverables
 from cg.models.rnafusion.rnafusion import CommandArgs
-from cg.store.models import Sample
 from cg.utils import Process
+from cg.models.deliverables.metric_deliverables import (
+    MetricsBase,
+    MultiqcDataJson,
+)
+from cg.store.models import Case
+from cg.io.json import read_json
+from cg.io.controller import ReadFile, WriteFile
+from cg.models.deliverables.metric_deliverables import (
+    MetricsDeliverablesCondition,
+)
+from cg.exc import CgError, MetricsQCError
+from cg.constants.tb import AnalysisStatus
+from cg.store.models import Sample
```

Comment:
There are some unused imports here

Reply:
Also in `rnafusion.py` and `taxprofiler.py` files 😜

Reply:
Removed those.